hivemq-edge/src/frontend/src/extensions/datahub/components/nodes/ClientFilterNode.tsx

```diff
@@ -18,25 +18,23 @@ export const ClientFilterNode: FC<NodeProps<ClientFilterData>> = (props) => {
       <NodeWrapper route={`node/${type}/${id}`} {...props}>
         <HStack>
           <VStack>
-            <Text> {t('workspace.nodes.type', { context: type })}</Text>
+            <Text data-testid={'node-title'}> {t('workspace.nodes.type', { context: type })}</Text>
           </VStack>
         </HStack>
-        <VStack ml={6}>
+        <VStack ml={6} data-testid={'node-model'}>
```

Comment:
suggestion(non-blocking): small little change on the attribute definitiona

```suggestion
        <VStack ml={6} data-testid="node-model">
```